File: android/app/src/main/java/com/qrcode_scanner/components/qrview/QrViewManager.java
package com.qrcode_scanner.components.qrview;

import com.dlazaro66.qrcodereaderview.QRCodeReaderView;
import com.facebook.react.uimanager.SimpleViewManager;
import com.facebook.react.uimanager.ThemedReactContext;
public class QrViewManager extends SimpleViewManager<QRCodeReaderView> {

    public static final String REACT_CLASS = "QRView";

    @Override
    public String getName() {
        return REACT_CLASS;
    }
    @Override
    public QRCodeReaderView createViewInstance(ThemedReactContext context) {
        return new QRCodeReaderView(context);
    }
}

File: android/app/src/main/kotlin/com/qrcode_scanner/components/qrview/QrViewManager.kt
package com.qrcode_scanner.components.qrview

import com.facebook.react.uimanager.SimpleViewManager
import com.facebook.react.uimanager.ThemedReactContext

class QrViewManager : SimpleViewManager<QrView>() {
    override fun getName(): String {
        return REACT_CLASS
    }

    public override fun createViewInstance(context: ThemedReactContext): QrView {
        return QrView(context)
    }

    companion object {
        val REACT_CLASS = "QRView"
    }
}
